Xamarin.Forms.Platform.iOS/CollectionView/ItemsViewController.cs

```diff
@@ -28,34 +24,29 @@ public abstract class ItemsViewController : UICollectionViewController
 		public ItemsViewController(ItemsView itemsView, ItemsViewLayout layout) : base(layout)
 		{
 			ItemsView = itemsView;
-			ItemsSource = CreateItemsViewSource();
```

Comment:
`CreateItemsViewSource()` is a virtual method and should not be called in the constructor. The debugger skips over this line and calls `ViewDidLoad()` before setting `ItemsSource` eventually.